pkg/az/config_test.go

```diff
@@ -0,0 +1,19 @@
+package az
+
+import (
+	"testing"
+
+	"get.porter.sh/mixin/az/pkg"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSetUserAgent(t *testing.T) {
+	pkg.Commit = "abc123"
+	pkg.Version = "v1.2.3"
```

Comment:
When you edit a global/package variable in a test, it will continue to be set for other tests which rarely is what you want. What I usually do is add a defer and then set the variable back to "".

Reply:
Thanks for catching that! I updated the TestMixin to always cleanup build information after each test